### PR TITLE
Day07

### PR DESCRIPTION
**Pull request by ArthurSoftvision1**

### CODE REVIEW

File: Curs04/src/views/evaluation.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        })}
+        ${Footer()}
+         `;
+}
+
+function EvaluationsTableHeader(options) {
+
+    const headings = [];
+
+    options.headings.forEach(function(el) {
```
Comment (s-timofte) [inline Curs04/src/views/evaluation.js:46]: So you have a sequence that generates an array containing elements based on the elements of another array. Can you think of any other way to achieve this other than using forEach?

File: Curs04/src/views/evaluation.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            ${EvaluationsTableHeader({
+                headings: ['Nume','Tehnologie','Nivel'
+            ]
+        })}
+            ${EvaluationsTableBody({
+                items: options.items
+            })}
+        </table>
+    </section>
+    `
```
Comment (s-timofte) [inline Curs04/src/views/evaluation.js:105]: your linter is not going to like something here :)

File: FirstProject/assets/evaluation.less [hunk 3]
```diff
@@ -74,19 +98,15 @@ table tr th {
   margin-top: 120px;
 }
 
-.col-md-4 {
```
Comment (s-timofte) [inline FirstProject/assets/evaluation.less:77]: bootstrap? :)

**Review by s-timofte — APPROVED** [review]
A next step would also be to clean up the html files and to transition everything to js

File: FirstProject/assets/evaluation.less (hunk 3 above)
Comment (ArthurSoftvision1) [reply FirstProject/assets/evaluation.less:77]: Initially was bootstrap, but I removed it.
eval

File: Curs04/src/views/evaluation.js (hunk 2 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/evaluation.js:105]: Better now? :)
eslint

File: Curs04/src/views/evaluation.js (hunk 1 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/evaluation.js:46]: Yes. We can use the map function and we can make an array with the header options.

With the map function we can go into the array and to display every option.

**Pull request merged**